lib/less/runalyze-style.css

```diff
@@ -5161,8 +5160,8 @@ td.datepickerDisabled a:hover {
 }
 @font-face {
   font-family: "iconvault";
-  src: url("../iconvault-forecast/../iconvault-forecast/iconvault_forecastfont.eot");
-  src: url("../iconvault-forecast/../iconvault-forecast/iconvault_forecastfont.eot?#iefix") format("embedded-opentype"), url("../iconvault-forecast/../iconvault-forecast/iconvault_forecastfont.woff") format("woff"), url("../iconvault-forecast/../iconvault-forecast/iconvault_forecastfont.ttf") format("truetype"), url("../iconvault-forecast/../iconvault-forecast/iconvault_forecastfont.svg#iconvault") format("svg");
+  src: url("iconvault_forecastfont.eot");
+  src: url("iconvault_forecastfont.eot?#iefix") format("embedded-opentype"), url("iconvault_forecastfont.woff") format("woff"), url("iconvault_forecastfont.ttf") format("truetype"), url("iconvault_forecastfont.svg#iconvault") format("svg");
```

Comment:
Do these icons still work? Seems odd to me (although the previous one is not perfect as well ...).